p/scripts/main.js

```diff
@@ -1313,9 +1318,6 @@ function init_simple_scrollbar() {
 var scrollTimeout;
 function init_sticky_sidebar(){
 	$sidebar = $('#sidebar');
-	if ($sidebar.length < 1) {
```

Comment:
This change was lost as well. I will check in the newest version to re-apply some of my changes.

Reply:
Sorry again for the headaches I seem to have caused!

Reply:
No worries :-)

Reply:
Should be fixed with https://github.com/FreshRSS/FreshRSS/pull/2117/commits/ef0c7ed367664282bb055f57bbba63855ca9b93a